### ISSUE TITLE
Chemical substances that "down regulate" STK11 - One ARS Actor Responds

### ISSUE THREAD

**Issue by sstemann** [NCATSTranslator/testing#28]
**Query**:  [Down Regulators Predicate Array STK11 (HGNC11389)(https://github.com/NCATSTranslator/testing/blob/main/ars-requests/not-none/stk11downregulators.json)
**PK**:  3/26 5b6ccd27-cc3a-46eb-8427-10f865c1de32
[**Results Tracking Sheet**](https://docs.google.com/spreadsheets/d/1Y2yfUFl2SIgv5KkyxdQB94UE3HtVdJNJ7KdnwQgZcYI/edit?usp=sharing)

![image](https://user-images.githubusercontent.com/38321826/112680734-1ec3c980-8e44-11eb-9722-21ff1a786776.png)

**Responses From**
ARAX (Expander Agent)

**Comment by kevinxin90** [comment]
@sstemann We use **entity_negatively_regulates_entity** for this specific relationship in BTE. This is also the predicate defined in [BioLink model](https://github.com/biolink/biolink-model/blob/master/biolink-model.yaml#L2516). So if you add this predicate in your query, you should be able to get results from BTE.
```
{
  "message": {
    "query_graph": {
      "edges": {
        "e01": {
          "object": "n0",
          "predicate": ["biolink:prevents",
            "biolink:negatively_regulates",
            "biolink:decreases_secretion_of",
            "biolink:decreases_secretion_of",
            "biolink:decreases_transport_of",
            "biolink:decreases_activity_of",
            "biolink:decreases_synthesis_of",
            "biolink:decreases_expression_of",
            "biolink:increases_degradation_of",
            "biolink:entity_negatively_regulates_entity",
            "biolink:disrupts",
            "biolink:directly_negatively_regulates",
            "biolink:inhibits",
            "biolink:inhibitor",
            "biolink:channel_blocker",
            "biolink:disrupts",
            "biolink:may_inhibit_effect_of"
          ],
          "subject": "n1"
        }
      },
      "nodes": {
        "n0": {
          "id": "HGNC:11389"
        },
        "n1": {
          "category": "biolink:ChemicalSubstance"
        }
      }
    }
  }
 }
```

**Comment by marcdubybroad** [comment]
Genetics provider does not contain ChemicalSubstance as a data category, so the trapi will return 0 results.

**Comment by pg427** [comment]
For Explanatory Agent, these are instances of queries that are not yet supported. We plan on implementing error codes shortly, so we anticipate that results like these will be less ambiguous in the near future. Of course, the Explanatory Agent will continue expanding its supported query space as we continue developing, so there should be fewer empty results in the coming months.

**Comment by yakaboskic** [comment]
CHP does not currently support any of these predicates.

**Comment by sstemann** [comment]
thank you! @kevinxin90 @andrewsu I re-ran with the updated predicate e27f42e1-667a-4280-b3fc-341ad95a7d04 BTE returns results

**Comment by sstemann** [comment]
> CHP does not currently support any of these predicates.

@yakaboskic is there a predicate that we could include that would connect chemical substances to STK11?

**Comment by andrewsu** [comment]
@sstemann On the subject of predicates, it looks like the list of predicates in the query are not all predicates in the Biolink Model. I was expecting all of them to be listed in https://biolink.github.io/biolink-model/docs/predicates.html, but this is what I found:

```
           "predicate": ["biolink:prevents",   #yes
            "biolink:negatively_regulates",    #no
            "biolink:decreases_secretion_of",  #yes
            "biolink:decreases_secretion_of",  #yes
            "biolink:decreases_transport_of",  #yes
            "biolink:decreases_activity_of",  #yes
            "biolink:decreases_synthesis_of",  #yes
            "biolink:decreases_expression_of",  #yes
            "biolink:increases_degradation_of",  #yes
            "biolink:negatively_regulates__entity_to_entity",    #no
            "biolink:disrupts",  #yes
            "biolink:directly_negatively_regulates",    #no
            "biolink:inhibits",    #no
            "biolink:inhibitor",    #no
            "biolink:channel_blocker",    #no
            "biolink:disrupts",  #yes
            "biolink:may_inhibit_effect_of"    #no
          ],
```

Can you clarify the source of your predicates?

**Comment by MarkDWilliams** [comment]
@andrewsu I believe we pulled those predicates from a list that Unsecret had been using to approximate the concept of "down regulates" across the KGs they consume, but it's definitely possible that some of those terms are from previous versions of Biolink or were KG specific.  Apologies if this caused confusion.

**Comment by webyrd** [comment]
Yes, not all of the predicates in the KGs we have are in Biolink,
unfortunately.

**Comment by yakaboskic** [comment]
> 
> 
> > CHP does not currently support any of these predicates.
> 
> @yakaboskic is there a predicate that we could include that would connect chemical substances to STK11?

@sstemann the predicate that we support that we currently support between chemical substance/drug to gene is "interacts_with." However, we don't have STK11 in our KP we do have: STK33, STK3, STK38L, STK38, STK32A, STKLD1, , STK32B, STK4, STK26, STK40. STK35. STK17B. STK31. STK24. STK10.

**Comment by sstemann** [comment]
update 4/20 PK: 92b20995-e3c8-4fa5-8306-d828c9c6bb53
![image](https://user-images.githubusercontent.com/38321826/115445773-4ceabe00-a1e4-11eb-9fc6-3712a9d75118.png)

**Comment by colleenXu** [comment]
I ran the updated query below through the ARS, PK: fca93057-3e46-401a-8fb3-c66597b146a6

![Screen Shot 2021-09-22 at 1 00 53 PM](https://user-images.githubusercontent.com/43731687/134413479-2900fa34-df1a-4014-b123-c32a146d416f.png)

The updated query could be something like this (it's missing the workflow section but the rest is TRAPI v1.2):
```
{
    "message": {
        "query_graph": {
            "nodes": {
                "n0": {
                    "ids": ["UniProtKB:Q15831", "HGNC:11389"],
		            "categories":["biolink:Gene", "biolink:Protein"]
                },
                "n1": {
                    "categories": ["biolink:ChemicalEntity"]
                }
            },
            "edges": {
                "e0": {
                    "subject": "n0",
                    "object": "n1",
                    "predicates":["biolink:entity_negatively_regulated_by_entity"]
                }
            }
        }
    }
}
```
